### DISCUSSION TITLE
Where to find examples of Azure DevOps pipelines for different platforms when single Uno project is used?

### DISCUSSION THREAD

**Discussion by Juriyx** [unoplatform/uno#16695] [Q&A]
Hello. I can't find simple and clear examples of Azure DevOps pipelines for Wasm, WinUI, Skia, Android, iOS and Mac Catalyst.
I tried to adapt the "Uno platform app" template in Visual Studio 2022 to my Wasm pipeline but it didn't build.

I got this error:
`C:\Users\VssAdministrator\.nuget\packages\microsoft.windowsappsdk\1.5.240404000\buildTransitive\Microsoft.WindowsAppSDK.SelfContained.targets(36,9): Error : The platform 'AnyCPU' is not supported for Self Contained mode.`

Help me, please

**Comment by agneszitte** [comment]
Hi @Juriyx 
We currently have this issue that we are tracking https://github.com/unoplatform/uno/issues/16662 for the `Error : The platform 'AnyCPU' is not supported for Self Contained mode.` 
For a pipeline you will need to have something similar: https://github.com/unoplatform/Uno.Samples/commit/81817c257bae7ac3fea7cb945bbdd53973f7795a
cc @jeromelaban

**Reply by jeromelaban** [reply]
@Juriyx you can update to the latest uno.sdk which will fix the error message. For the rest, documentation for publishing is still being built.

**Reply by digger73** [reply]
> @Juriyx you can update to the latest uno.sdk which will fix the error message. For the rest, documentation for publishing is still being built.

Any update on this at the current time ?

**Reply by jeromelaban** [reply]
The documenation was updated for publishing a while back: https://platform.uno/docs/articles/uno-publishing-windows.html
